lib/components/molecules/QuoteMedia/QuoteMedia.mock.js

```diff
@@ -0,0 +1,20 @@
+import Blockquote from '../../atoms/Blockquote/Blockquote';
+
+const defaultQuoteMedia = {
+  className: 'blockquote-container',
+  quoteClassName: '',
+  HeadingType: 'h2',
+  title: 'Lorem ipsum dollor',
+  quoteAuthorProp: 'Lorem ipsum',
+  quoteProp: 'Lorem ipsum dollor sit amet',
+  hasContent: false,
+  quoteContent: (quoteClassName, quoteAuthorProp, quoteProp) => (
```

Comment:
(quoteClassName, quoteAuthorProp, quoteProp) is wrong pattern NEVER do it 

This is no longer needed but even if some where you need to pass multiple parameter Create an Object

Reply:
fixed